js/ccf-app/test/polyfill.test.ts

```diff
@@ -510,21 +510,29 @@ describe("polyfill", function () {
           const jwk = ccf.crypto.pubPemToJwk(pair.publicKey);
           assert.equal(jwk.kty, "EC");
           assert.notEqual(jwk.kid, my_kid);
+          const pem = ccf.crypto.pubJwkToPem(jwk);
```

Comment:
nit: Maybe change `describe("pemToJwk", ...` to `describe("pemToJwk and jwkToPem", ...` or something like that?
I can put comment only on changed lines, so I put it here.